### ISSUE TITLE
Add the Task Location list as a menu item (FT UI).

### ISSUE THREAD

**Issue by bsaiken** [FWAJL/FieldWorkManager#1075]
I set this up in an ad-hoc way, but it will not work, except it is good as a mockup of what we need.  

It turns out, for the mobile map viewed on a smartphone, we must have the map and the location list separated.

So, properly remove the Location list from `/mobile/map`.  

Also, properly remove the Map from the `/mobile/loc_list`.  Right now the map is there, but I just added line 123 to `app-mobile-style.css` to hide it.  

It's a little tricky, I think, because when scrolling over a map icon, the location list icon highlights and vise-verse.  

All mouseover (and similar) and drag behaviors need to be removed from the Mobile UI (see Issue #1097 and #1066).

**Comment by WebDevJL** [comment]
@bsaiken 

We will need to discuss more about this issue. I am not sure what you want to achieve. Plus, your commit seems to be a lot of copy/paster = harder to maintain code.

**Comment by bsaiken** [comment]
@FWAJL 
We can (and will) fix this later.  For now, the work that I have done has creates the desired view.

**Comment by damirius** [comment]
@bsaiken @FWAJL 

I've separated loc_list page from map. Map element is totally removed from that page and I've moved ajax call along with some other functions (for info windows) to mobile_manager.js. I've also hidden some buttons which we don't need on this list, like directions/zoom since we don't use a map here. User still should be able to upload or remove photos from the locations and edit description, longitude and latitude. Collect data button should also show in task location info windows.

@FWAJL We can try to merge this with code @gsouvik done for issue #1079 if everything is ok with the list

Commit is on feature_damir branch
